implementations/rust/ockam/ockam_command/tests/bats/portals.bats

```diff
@@ -228,3 +228,17 @@ teardown() {
   run_success "$OCKAM" tcp-outlet create --at /node/blue --to 127.0.0.1:5000
   run_success curl --head --retry-connrefused --retry 2 --max-time 10 "127.0.0.1:$port"
 }
+
+@test "portals - local inlet and outlet in reverse order" {
+  port="$(random_port)"
+
+  run_success "$OCKAM" node create n1
+  run_success "$OCKAM" node create n2
+
+  run_success "$OCKAM" tcp-inlet create --at /node/n2 --from "127.0.0.1:$port" --to /node/n1/service/outlet
+  run_success "$OCKAM" tcp-outlet create --at /node/n1 --to 127.0.0.1:5000
+
+  sleep 5
+
+  run_success curl --fail --head --max-time 20 "127.0.0.1:$port"
```

Comment:
I think the n1 must be created after the inlet, otherwise it will connect right away.